src/main/java/com/dynatrace/openkit/DynatraceOpenKitBuilder.java

```diff
@@ -39,6 +39,17 @@
      * @param deviceID      unique device id
      */
     public DynatraceOpenKitBuilder(String endpointURL, String applicationID, long deviceID) {
+        this(endpointURL, applicationID, Long.toString(deviceID));
+    }
+
+    /**
+     * Creates a new instance of type DynatraceOpenKitBuilder
+     *
+     * @param endpointURL   endpoint OpenKit connects to
+     * @param applicationID unique application id
+     * @param deviceID      unique device id, truncated to 250 characters
```

Comment:
the provided string is not truncated. I'd rather add a remark that only up to 250 chars are used